packages/web/docs/src/pages/get-started/single-project.mdx

```diff
@@ -0,0 +1,295 @@
+import NextImage from 'next/image'
+import { Callout, Steps } from '@theguild/components'
+import cdnTokenImage from '../../../public/docs/pages/guides/cdn_token.png'
+import historyImage from '../../../public/docs/pages/guides/history.png'
+import tokenImage from '../../../public/docs/pages/guides/token.png'
+
+Once you've created a Hive project of type **Single Schema**, you can simply push your GraphQL
+schema to the registry. This guide will guide you through the basics of schema pushing, checking and
+fetching.
+
+<Steps>
+
+### Your GraphQL Schema
+
+If you already have a GraphQL schema from your project, you can use it directly. The Hive CLI can
+read GraphQL schemas from `.graphql` files in your local filesystem.
+
+<Callout type="info">
+  **Using code-first approach to write your GraphQL schema?**
+  
+  Please refer to the [Code-First Guide](/guides/code-first) to learn how to use the Hive with
+code-first approach.
+
+</Callout>
+
+If you don't have a schema, you can use the following schema in this guide. Copy the contents into a
+`schema.graphql` file and save it to your filesystem:
+
+```graphql filename="schema.graphql" showLineNumbers
+type Query {
+  viewer: ViewerQuery!
+  user(id: ID!): User!
+}
+
+type ViewerQuery {
+  user: User!
+  role: UserRole!
+}
+
+type User {
+  id: ID!
+  name: String!
+  age: Int!
+}
+
+enum UserRole {
+  USER
+  ADMIN
+}
+```
+
+### Hive CLI Access Token
+
+As you may have noticed, Hive has created three targets under your project: `development`,
+`staging`, and `production`. This guide will use the `development` target to explore the features
+Hive offers.
+
+To begin, select the `development` target under your Hive project and then choose the **Settings**
+tab. On the Settings screen, you can manage your target's settings and access tokens.
+
+Under the **Registry Access Tokens** section, click the **Generate new token** button. Give your
+access token a name and select **Schema Check & Push** from the list of token presets:
+
+<NextImage
+  alt="Create Registry Token Form"
+  src={tokenImage}
+  className="mt-10 max-w-md drop-shadow-md"
+/>
+
+Click on **Generate Token** button and you should get your CLI token created, with permissions to
+publish GraphQL schemas.
+
+Make sure to copy your token and keep it safe. **You won't be able to see it again.**
+
+<Callout type="info" emoji="ℹ️">
```

Comment:
```suggestion
<Callout type="info">
```
You can already have Info icon with `type="info"` that much beauty than info emoji